app/models/offsite_link.rb

```diff
@@ -26,4 +25,29 @@ def humanized_link_type
   def to_s
     title
   end
+
+private
+
+  def url_is_gov_wales?(host)
+    return false if host.nil?
+    host =~ /gov\.wales$/
+  end
+
+  def url_is_gov_uk?(host)
+    return false if host.nil?
+    host =~ /gov\.uk$/
+  end
+
+  def url_is_whitelisted?(host)
+    whitelisted_hosts = [
+      "flu-lab-net.eu",
+      "tse-lab-net.eu",
+    ]
+
+    whitelisted_hosts.each do |whitelisted_host|
```

Comment:
Fixed in 25729d9. Also slimmed the preceding methods down a bit. The `nil` check is redundant now as `nil =~` is valid.